### PR TITLE
storage: add Virtio-blk to proto

### PR DESCRIPTION
**Pull request by glimchb**
Signed-off-by: Boris Glimcher <Boris.Glimcher@emc.com>

### CODE REVIEW

File: storage/proto/frontend.proto [hunk 1]
```diff
@@ -72,6 +80,15 @@ message NVMeNamespace {
     string authentication = 13;
 }
 
+message VirtioBlk {
+    int64 id = 1;
```
Comment (mestery) [inline storage/proto/frontend.proto:84]: Eventually, do you want to change these to uuids?

File: storage/proto/frontend.proto [hunk 2]
```diff
@@ -224,7 +241,59 @@ message NVMeNamespaceStatsRequest {
...
+    // Intentionally empty.
+}
+
+message VirtioBlkDeleteRequest {
+    int64 subsystem_id = 1;
+    int64 controller_id = 2;
+}
+
+message VirtioBlkDeleteResponse {
+    // Intentionally empty.
```
Comment (mestery) [inline storage/proto/frontend.proto:263]: For these, do you want to return the Id values of the object you changed?

**Review by mestery — APPROVED** [review]
Looks fine, you can address my comments if you want.

File: storage/proto/frontend.proto (hunk 1 above)
Comment (glimchb) [reply storage/proto/frontend.proto:84]: YES!!!!!!!!!

File: storage/proto/frontend.proto (hunk 2 above)
Comment (glimchb) [reply storage/proto/frontend.proto:263]: YES!!!!!!!!!

**Pull request merged**